README.md

```diff
@@ -1 +1,74 @@
-# codepen-blog-post
+We all know that our beloved Javascript language is somewhat insecure, so errors at runtime (hello 'undefined' is not an object) occur very frequently when systems grow in complexity.
+
+One way to improve security in that regard is by declaring a factory function that allows us to create and use our plain old javascript objects in a more secure way using ES6 proxies.
+
+```
+const proxyHandler = {
```

Comment:
Yo pondría el cómo se usa el código primero (el resultado de los console.logs) y después un "what's under the hood?" con este código y su explicación.

Que el bloque de código sea para los interesados, y el resultado los haga interesarse.

Reply:
como dar vuelta los bloques de codigo dices tu ? (onda poner el uso que sale abajo primero y despues esto?)